app/models/ontology.rb

```diff
@@ -130,6 +131,11 @@ def iri_for_child(child_name)
     child_name.include?("://") ? child_name : "#{iri}?#{child_name}"
   end
 
+  def locid_for_child(child_name)
+    child_name = child_name[1..-2] if child_name[0] == '<'
+    child_name.include?("://") ? child_name : "#{locid}//#{child_name}"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.